rootfs/api/models.py

```diff
@@ -203,7 +203,7 @@ def _get_command(self, container_type):
             # and run arbitrary commands on the host
             # FIXME: remove slugrunner's hardcoded entrypoint
             release = self.release_set.latest()
-            if release.build.dockerfile or not release.build.sha:
+            if release.build.dockerfile:
```

Comment:
I think this conditional is still required for `deis pull` workflows, since it won't have a Dockerfile or a sha present.